packages/neptune-extension/src/common/utils/checkout.ts

```diff
@@ -32,7 +32,7 @@ export async function fetchCheckpointOptions(notebookId?: string) {
 
   const { entries } = await leaderboardClient.api.listCheckpoints({ notebookId });
   return entries.map(entry =>
-    [ entry.id, (new Date(entry.creationTime)).toLocaleDateString() ]
+    [ entry.id, entry.name ? entry.name : (new Date(entry.creationTime)).toLocaleString() ]
```

Comment:
UX suggested format:
`entry.name|(unnamed) - formatDate(entry.creationTime)`
where formatDate is the same function we use in app.